pkg/apis/extensions/validation/validation.go

```diff
@@ -447,6 +447,29 @@ func ValidateIngressUpdate(oldIngress, ingress *extensions.Ingress) errs.Validat
 	return allErrs
 }
 
+func ValidateIngressStatusUpdate(ingress, oldIngress *extensions.Ingress) errs.ValidationErrorList {
```

Comment:
Godoc please (too many public functions in this file don't have one, but at least all the Ingress ones do, so victory!)